packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -442,11 +445,21 @@ handler.scanForMissingPullRequests = async function scanForMissingPullRequests(
       const ownerAndRepoArray = issue.repository_url.split('/');
       const owner = ownerAndRepoArray[ownerAndRepoArray.length - 2];
       const repo = ownerAndRepoArray[ownerAndRepoArray.length - 1];
+      let baseBranch;
+      try {
+        baseBranch = (
+          await github.pulls.get({owner, repo, pull_number: issue.number})
```

Comment:
might be worth waiting to see if there's a known configuration where this exception happens, before we protect against it.

Reply:
Probably it's better to fetch the baseBranch later in `checkForBranchProtection`.

Here is the reason.

When you rename the default branch on the github UI, github will modify existing PRs' base branch to the new default branch.

Reply:
I also like that it's less lines of code :)